documentation/Recipe-Language.md

```diff
@@ -0,0 +1,192 @@
+# Recipe language
+
+The core means of using the Tombolo Digital Connector is to create a data export recipes, describing the data or models that we want to get out of the system, without describing how to generate the data. The recipe language is expressed in the well-known JSON format.
+
+## Data export recipe
+A data export recipe consists of two parts:
+
+* **Dataset**: is a description of the data to be exported. The format of the dataset recipe is explained in the [Dataset recipe](#dataset-recipe) section below.
```

Comment:
I would be consistent with the **markdown** style used.

For example here `**` is used here but in other places `__` is used:

- **Bold using \*\*** (`**Bold using asterix**`)
- __Bold using \_\___ (`__Bold using underscore__`)